### PR TITLE
  **Fix:** adjust plus button for tabs

### PR DESCRIPTION
**Pull request by stereobooster**
# Summary

Screenshot 2019-08-13 at 14 05 24
Screenshot 2019-08-13 at 14 05 41

# Related issue

UI-125

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

**Comment by TejasQ** [comment]
This looks a bit broken? 
image

**Comment by stereobooster** [comment]
@TejasQ addressed

File: src/Tabs/Tabs.styled.ts [hunk 1]
```diff
@@ -15,11 +16,17 @@ export const Container = styled.div`
   padding-top: 1px;
 `
 
+export const PlusWrapper = styled.div`
+  display: flex;
+  max-width: 100%;
```
Comment (TejasQ) [inline src/Tabs/Tabs.styled.ts]: ? why

File: src/Tabs/Tabs.styled.ts [hunk 2]
```diff
@@ -119,8 +126,8 @@ export const TabButton = styled(SectionHeader, {
     transparent ? theme.color.background.light : theme.color.background.lighter};
   margin: 0;
   padding: 0;
-  width: ${buttonWidth}px;
-  min-width: ${buttonWidth}px;
+  width: ${({ transparent }) => (transparent ? plusButtonWidth : buttonWidth)}px;
```
Comment (TejasQ) [inline src/Tabs/Tabs.styled.ts]: `isTransparent` would make the intent of this prop a little more clear I think

Comment (TejasQ) [reply src/Tabs/Tabs.styled.ts]: I also don't understand the logic?

If transparent, use the plus button's width. But why? What?

**Comment by stereobooster** [comment]
@TejasQ fixed

**Pull request merged**